regions_www/index.html

```diff
@@ -62,9 +62,8 @@
           <p> Welcome to the Propensity to Cycle Tool (PCT) for England, which provides an evidence base to inform cycling investment.</p>
           <p>To run the PCT, first hover over a region on the map on the left to see the regional potential, then click on the region.</p>
           <p>For an introduction to the tool, its use, and the scenarios, please watch the video above.</p>
-          <p><strong>What's new :</strong> <a href="https://blog.pct.bike/2017/03/21/lower-level-super-output-area-route-network-now-up/", target = "_blank"> LSOA layer</a>;
-          <a href="http://www.cedar.iph.cam.ac.uk/resources/evidence/eb-14-englands-cycling-potential/", target = "_blank">Local-authority evidence briefing</a>
-            </p>
+          <p><strong>What's new :</strong> <a href="https://blog.pct.bike/2017/09/05/new-features/" target = "_blank">Cross-border flows, downloads, Wales & more</a>
```

Comment:
Really we should use `&amp;` instead of `&`. It works but I think its considered a bit safer as `&` is a keyword in html.